wiki/osu!academy/en.md

```diff
@@ -1,6 +1,5 @@
 ---
 needs_cleanup: true
-outdated: true
 ---
 
 <!-- TODO: new videos and branding -->
```

Comment:
if this TODO is fulfilled (I guess it is?), please remove the commentary as well as the whole preamble, which is first 3 lines, and a blank line after it

Reply:
Do you want me to remove the cleanup tag as well?

Reply:
I believe TicClick is referring to the fact that this part...

> We hope that new players will find these videos informative, and help ease the arguably steep learning curve associated with the game.
If you have any questions or comments, please forward them to [ztrot](https://osu.ppy.sh/users/6347); he is the primary and almost sole owner of the osu! academy and its channel.
>
> [Now with a trailer](https://www.youtube.com/watch?v=z5gy34k3RI0&feature=c4-overview&list=UUMeRgqzTfC5ja40B6kM6pdg "YouTube").

needs to be either removed or adjusted to be in-line with writing standards, then yes, you can remove the TODO and cleanup tag

Reply:
to me, the TODO, as well as the `needs_cleanup` tag, is about adding missing series. I didn't read into the whole article and suggested that #5884 and #5888 covered it completely. if that's not the case, then please only remove the TODO and keep `needs_cleanup`